src/app/heroes/heroes.component.spec.ts

```diff
@@ -37,4 +50,13 @@ describe('Heroes Component', ()=> {
     expect(heroesCmp.heroes[1]).toBe(HEROES[2]);
   })
 
+  it('should call delelteHero with the correct hero', ()=> {
+    heroesCmp.heroes = HEROES;
+    mockHeroSvc.deleteHero.and.returnValue(of(true));
```

Comment:
Just one advise that will make things easier for you. When creating a mockedServiceSpy, add types to have intellisense and be able to walk through all the methods you can use in your tests

like this: 
```
let valueServiceSpy: jasmine.SpyObj<ValueService>;
let stubValue = 'stub value';

valueServiceSpy = jasmine.createSpyObj('ValueService', ['getValue']);
valueServiceSpy.getValue.and.returnValue(stubValue);

```

Look here:
![image](https://user-images.githubusercontent.com/35868173/110475037-0dab5680-80e9-11eb-861a-8ff93d743d62.png)